eq-author-api/middleware/auth.test.js

```diff
@@ -113,5 +113,38 @@ describe("auth middleware", () => {
         expect(next).toHaveBeenCalled();
       });
     });
+
+    describe("token containing =", () => {
```

Comment:
I was expecting a small test with a token containing equals to be added to the previous describe - you can just redefine the mock to return the new bearer in the test. Thats all the logic that has been added AFAICT.

Reply:
it should be a new scenario if the mocks are changing shouldn't it?